lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -159,6 +161,106 @@ static bool isZeroBitType(Type type) {
   return false;
 }
 
+// structs and logic/reg/wire are base types that arrays are built up from
+// and they are printed independently of their context.  Integers and arrays and
+// unpacked arrays create non-trivial array bound definitions on each side of
+// the name.  So we break printing up into those three parts: baseType
+// (struct,logic,reg,wire), packed dimensions (integer bitwidth, packed array
+// width), name (left to caller), and unpacked dimensions.
+
+// Print out the packed array subscripts and integer bitwidth after a wire/port
```

Comment:
nit: MLIR has the convention of code comments for the function (function docs) are `///` comments. CIRCT is replicating that standard.